Boilerplates/Application/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
If I'm reading this right, every Application created will have this license - we can't enforce that. So we should therefor delete this file

Reply:
Nope, only the stuff in the Content folder is created by the boilerplate